apps/web/src/app/[orgShortcode]/settings/org/users/invites/_components/invite-modal.tsx

```diff
@@ -83,6 +83,9 @@ export function InviteModal() {
       form.reset();
       void invalidateInvites.invalidate();
       setOpen(false);
+    },
+    onError: (error) => {
```

Comment:
don't set a `onError` callback unless you need to do any specific work, if you catch the error here it would not go to the global handler which would render out the error toast

Reply:
ok sol how would you want me to handle this?

Reply:
![Capture](https://github.com/user-attachments/assets/1fb617fe-bd17-4ab1-b3d3-3816fab57cb7)


Would it be ok if I simply remove the error part inside the modal. or we will go with the above approch?

Reply:
ah it is being handled in a modal, you can just do this

```ts
onError: () => void 0,
````

Reply:
Yup works! Have updated it.